README.md

```diff
@@ -35,3 +35,37 @@ Prefix: `p!`
 - `help`
 - `ping` will return `pong`
 - `challenge OPPONENT`
+
+# Testing
+
+## Unit Tests
+
+```
+docker-compose run bot sh ./scripts/test.sh
+```
+
+## Integration Tests
+### Setup
+1. Create another bot. This will be the "tester".
+2. Make sure these settings are enabled under the `Bot` section.
+
+![Both Privileged Gateway Intents enabled](docs/img/ci-runner-required-intents.png)
+
+3. Run `docker-compose run bot sh ./scripts/integration-test.sh` to create the config file at `tests/config`
+4. Fill out the fields in the newly generated `tests/config`
+
+### Running the Integration Tests
+1. Make sure Brock is running
+```
+docker-compose up -d
+```
+2. Run the tests
+```
+docker-compose run bot sh ./scripts/integration-test.sh
```

Comment:
Should we be using bash instead of sh since that's what we have in the script shebangs? or should we update the shebangs?

Reply:
I just tried it and the `sh` is not required.